### PR TITLE
PAY-776: Onboarding Magento Payments topic

### PR DESCRIPTION
**Pull request by shrielenee**
## Purpose of this pull request

This pull request (PR) adds an Onboarding Payments topic to the Payments documentation integration branch, `ss-payments-merch`.

This topic is a first iteration on a comprehensive onboarding topic that will walk the merchant through all the needed steps for onboarding Magento Payments for the Magento store.

**Jira**: https://jira.corp.magento.com/browse/PAY-776
**Preview**: 55/user-guide/magento-payments/onboarding-payments.html

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- user-guide/magento-payments/onboarding-payments.html

## Links to Magento source code or PRs

- ...
- ...

### CODE REVIEW

**Review by nofuss — COMMENTED** [review]
Several comments on the initial list of steps.

File: src/magento-payments/onboarding-payments.md
Comment (nofuss) [inline src/magento-payments/onboarding-payments.md:92]: probably don't want to include the password in the screenshot

**Review by nofuss — COMMENTED** [review]
minor

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:9]: SaaS is not a branded term in general, though we use it that way in the Admin. The first time an acronym is introduced on a page it is fully qualified and includes the acronym in parentheses. After that, only the acronym is used. This is a standard convention for documentation.

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:10]: What about "Onboard and set up the sandbox service"?

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:92]: Yes, you are right. I have a separate story to rejigger all screenshots as needed.

**Review by cherdman — APPROVED** [review]
Looks great, Shrie. Very clear and concise.

File: src/magento-payments/onboarding-payments.md
Comment (melnikovi) [inline src/magento-payments/onboarding-payments.md:5]: We are not going to have fraud prevention for EAP, should we clarify it or remove it?

File: src/magento-payments/onboarding-payments.md
Comment (melnikovi) [inline src/magento-payments/onboarding-payments.md:11]: The merchant can't process payments until he onboarded, not sure if it needs further clarifications.

File: src/magento-payments/onboarding-payments.md
Comment (melnikovi) [inline src/magento-payments/onboarding-payments.md:35]: Maybe better to call them API keys (they are called API and private keys on the extension configuration page)?

File: src/magento-payments/onboarding-payments.md
Comment (melnikovi) [inline src/magento-payments/onboarding-payments.md:102]: If we are configuring sandbox payments, should we mention enabling live payments?

File: src/magento-payments/onboarding-payments.md
Comment (melnikovi) [inline src/magento-payments/onboarding-payments.md:107]: We are enabling for websites technically, I wonder if we should use work websites instead of stores.

**Comment by melnikovi** [comment]
Left some minor comments, otherwise looks good.

File: src/magento-payments/onboarding-payments.md
Comment (melnikovi) [inline src/magento-payments/onboarding-payments.md:20]: We should probably clarify the part where the merchant is waiting for approval on the diagram. Maybe at least mention that it doesn't take a lot of time or the page might need to be refreshed (I think we will have to see what the experience will be).

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:5]: Good point, yes.

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:11]: I don't think it needs further clarification. These steps are listed in the order they should likely be completed. In the next two steps you have to complete merchant onboarding and active payments in live mode to start processing live payments. I think we're okay here.

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:20]: We do. In both the Sandbox onboarding and Live payment onboarding we have sections for that.
Screen Shot 2021-02-10 at 2 32 10 PM

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:102]: Good call.

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:107]: Yes!

File: src/magento-payments/onboarding-payments.md
Comment (melnikovi) [reply src/magento-payments/onboarding-payments.md:20]: I was wondering if we should mention in the documentation what do we expect from the merchant, for example how long the merchant has to wait. Like I said, we can see what the experience will be after we have it developed and come back to this.

**Comment by shrielenee** [comment]
@nofuss @melnikovi I have pushed my changes based on your review. Can I get one more approval for merging?

File: src/magento-payments/onboarding-payments.md
Comment (melnikovi) [inline src/magento-payments/onboarding-payments.md:102]: Hm, you can't enable both.

**Review by melnikovi — APPROVED** [review]

File: src/magento-payments/onboarding-payments.md
Comment (shrielenee) [reply src/magento-payments/onboarding-payments.md:102]: Ahh yes. Oversight--I will revise this.

**Pull request merged**
